pkgs/servers/home-assistant/custom-components/emporia_vue/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, buildHomeAssistantComponent
+, pyemvue
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "presto8";
+  domain = "emporia_vue";
+  version = "0.8.3";
+
+  src = fetchFromGitHub {
+    owner = "magico13";
+    repo = "ha-emporia-vue";
+    rev = "master";
+    hash = "sha256-lSxYg4nradJQVNBc6LMp4EHsXFM30AKf3ynqjznP2Lo=";
+  };
+
+  propagatedBuildInputs = [
+    pyemvue
+  ];
+
+  # The latest version of pyemvue on pypi.org is 0.18.0. But ha-emporia-vue
```

Comment:
We don't need such a big comment, we do this all the time

Reply:
Make sense, I'll trim it!